src/components/InputTodo.js

```diff
@@ -0,0 +1,29 @@
+import { FaCirclePlus } from 'react-icons/fa6';
+import PropTypes from 'prop-types';
+
+const InputTodo = ({ addTaskToState }) => {
+  const newTask = (e) => {
+    e.preventDefault();
+    const form = e.target;
+    const taskTitle = form.inputDescription.value;
+    if (taskTitle.length > 0) {
+      addTaskToState(taskTitle);
+      form.reset();
+    }
+  };
+
+  return (
+    <form id="form-input-todo" onSubmit={newTask}>
+      <div id="container-input-todo">
+        <input type="text" placeholder="Add todo..." id="inputDescription" />
+        <button type="submit" id="submit-new-task" aria-label="Add task"><FaCirclePlus /></button>
```

Comment:
The input is not validated for an empty value. I kindly suggest you handle that using ```required``` attribute.

Reply:
Hi again! @Gh-Reza Thank you so much for this review. Your recommendation is very correct. I have applied these changes, you can see them in the last commits.